src/edl/edl_initializer.cpp

```diff
@@ -0,0 +1,92 @@
+#include "edl_initializer.h"
```

Comment:
Logging is initialized by this time already right?

Reply:
Logging doesn't need to be initialized.

Reply:
I thought about why you use printf instead of logger here

Reply:
Forgot about my logger i guess 🥴